### ISSUE TITLE
support selfhosted music sites? (Ampache)

### ISSUE THREAD

**Issue by lachlan-00** [carlin-q-scott/browser-media-players#52]
Hey, i know it says request the site and give out the html but i was just wondering if this was domain dependent?

I maintain Ampache (github.com/ampache/ampache/) and something like this has actually been requested a lot.

if i did a request with the details for our web player would it be possible for it to work on any site domain? because it's selfhosted it could be anything including just an ip.
![image](https://user-images.githubusercontent.com/1305249/88448060-06e31280-ce7d-11ea-894f-9e2e23551e6e.png)

**Comment by carlin-q-scott** [comment]
Yes, the extension settings page has a place to configure custom domains for a select set of media players. Ampache could be added to that list.

**Comment by lachlan-00** [comment]
cool, thanks for replying. I'll document it on the ampache wiki once I work it all out.
